modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
+
+To subscribe to Redfish Events, create a BMCEventSubscription CRD.
```

Comment:
create a BMCEventSubscription CR.

It is `CR`, not `CRD`.

Reply:
Topic now in scope of https://github.com/openshift/openshift-docs/pull/41832.